src/config.cc

```diff
@@ -367,14 +367,16 @@ void Config::initFieldCallback() {
         if (!RocksDB.enable_blob_files) {
           return Status(Status::NotOK, errNotEnableBlobDB);
         }
-        return srv->storage_->SetColumnFamilyOption(trimRocksDBPrefix(k), v);
+        return srv->storage_->SetColumnFamilyOption(trimRocksDBPrefix(k),
+                                                    std::to_string(RocksDB.min_blob_size));
```

Comment:
don't need to change, right?
If not necessary, we should not dirty blame log which provide the clue to kvrocks evolution.

```suggestion
        return srv->storage_->SetColumnFamilyOption(trimRocksDBPrefix(k), v);
```